extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -347,6 +352,11 @@ function(_juce_get_platform_plugin_kinds out)
         list(APPEND result AU)
     endif()
 
+    # For now, only enabling LV2 builds on Linux
+    if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+        list(APPEND result LV2)
+    endif()
+
```

Comment:
Linux-only could be a good place to start, but I'm wondering if we could get away with enabling LV2 for all the platforms that VST3 is enabled for. If we were to do that, though, we'd have to set `JUCE_LV2_COPY_DIR` for macOS and Windows -- in my port, I figured I could more or less just copy-paste whatever the VST3 builds did. In that case, fixing the `_COPY_DIR`s for other platforms might be as easy as changing the the "VST3" to a "LV2".

Reply:
Yeah, I imagine it shouldn't be too hard to enable LV2 builds for all platforms, I just haven't done it yet since for the moment I'm only able to test LV2s on Linux.

Reply:
Sometime in the next couple days I intend to test out doing a Windows build, but I don't know if anyone will be able to check on macOS.